core/test/unit/utils_read_csv_spec.js

```diff
@@ -0,0 +1,55 @@
+/*globals describe, it*/
```

Comment:
I would like to keep the test folder structure. (`test/unit/utils/read-csv_spec.js`)
In my opinion server folder/file structure and test folder/file structure should be equal.

Here some advantages:
1. we don't have hundrets of files in the root folder of the unit test folder
2. for new developers/contributors its easier to find test files
3. ...